README.md

```diff
@@ -0,0 +1,64 @@
+# Nonlocal Gravity Wave Flux Prediction
+
+The code in this repository implements three classes of deep learning models to predict atmospheric gravity wave momentum fluxes, provided the atmospheric background conditions as input.
+
+## Models description
+The three kinds of models are:
+1. **M1:** A single column ANN which takes single column atmospheric variables as input and predicts the gravity wave momentum fluxes in the single column (pixel-to-pixel regression)
+2. **M2:** A regionally nonlocal CNN+ANN which takes a (2N+1)x(2N+1) horizontal columns of atmospheric variables as input and predicts the fluxes in the central single column (image-to-pixel regression)
+3. **M3:** A global attention UNet model which taked global horizontal maps of variables as input to predict fluxes over the whole horizontal domain (image-to-image regression)
+The three models are schematically depicted in the animation below
+
+
+
+![Schematic for the three models](https://amangupta2.github.io/images/icml_schematic.gif)
```

Comment:
thanks for adding diagrams to the `README.md`. This is a great way to provide more context

Reply:
I appreciate the pictures as well but find the gif transitions a little quickly for my taste. Consider just using the final frame as a static image?